Sources_v3/Controllers/ChannelController.swift

```diff
@@ -201,6 +233,10 @@ public extension ChannelControllerGeneric {
                        extraData: ExtraData.Channel,
                        completion: ((Error?) -> Void)? = nil)
     {
+        guard isChannelCreated == true else {
+            log.error("You cannot perform channel actions before its creation on backend.")
+            return
+        }
```

Comment:
We need to let users know this is not a valid operation. I'd suggest calling the completion with `ChannelDoesNotExist` error.